### ISSUE TITLE
move method not working on window load (working only after window resize or page refresh)

### ISSUE THREAD

**Issue by bini7a** [lukehaas/Scrollify#127]
I am trying to use move method on window load with the index but it doesn't work.

```
$(window).on('load', function(){
    $.scrollify.move(3);
});
```

I can trigger the resize event 

```
$(window).on('load', function(){
    $.scrollify.move(3);
}).resize();
```

Now, this works but  it scrolls to the defined section and goes back to the top section (0 index) and then only move again to the defined section.

**Comment by bini7a** [comment]
I just see there is also update method and tried that too. This works but the problem is same as stated above 'first goes to 0 index and then move to the defined section'.

**Comment by lukehaas** [comment]
The `afterRender()` callback is best suited to what you need.
Scrollify will always start from the first section unless the fragment identifier (hash #) is pointing to a different section.

**Comment by bini7a** [comment]
I also tried afterRender callback but problem persists as same as before.

**Comment by lukehaas** [comment]
That's right, Scrollify will always got to the first section unless you have a hash value that is pointing to a different section.

**Comment by bini7a** [comment]
It is not going to first section firstly, first it goes to the defined section and then goes to the first section and again goes to the defined section.

**Comment by lukehaas** [comment]
So you have a hash value that defines which section to start from and you're also using the move method after render to navigate away from that starting section?

**Comment by bini7a** [comment]
I don't have set hash value. Does it take first section automatically by the plugin?

**Comment by lukehaas** [comment]
Yes, if no hash value is set then it will default to the first section.

**Comment by bini7a** [comment]
Can I set hash value to none?

**Comment by lukehaas** [comment]
Yes, if you set the `sectionName` option to false.

**Comment by bini7a** [comment]
Oh! That is what I already done.

**Comment by lukehaas** [comment]
Then you can solve your issue by not setting it to false and Scrollify will then default to whichever section you want it to, by appending a hash value to the URL.
